components/Breadcrumb.vue

```diff
@@ -82,9 +82,8 @@ export default class Breacrumb extends Vue {
       case '/dashboard/': {
         let isStaff: boolean
         try {
-          const keycloakRoles = getKeycloakRoles()
-          isStaff = (keycloakRoles.includes('gov_account_user') || keycloakRoles.includes('staff'))
```

Comment:
Per RC discussion, government users get the same breadcrumbs as non-staff users.

Reply:
Can we continue this discussion in RC?

Reply:
Fixed! All we need in this component is the store getter `isSbcStaff`, which is now set in Dashboard.vue.